source/accessibility-statement.html.md.erb

```diff
@@ -0,0 +1,96 @@
+---
+title: Accessibility
+weight: 999
+hide_in_navigation: true
+---
+
+# Accessibility
+
+## Accessibility statement for data.gov.uk documentation
+
+This accessibility statement applies to the data.gov.uk technical documentation at [https://guidance.data.gov.uk/#data-gov-uk-technical-documentation](https://guidance.data.gov.uk/#data-gov-uk-technical-documentation).
+
+This website is run by the data.gov.uk team at the Government Digital Service (GDS). We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
+- change colours, contrast levels and fonts
+- zoom in up to 300% without problems
+- navigate most of the website using just a keyboard
+- navigate most of the website using speech recognition software
+- listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+We’ve also made the website text as simple as possible to understand.
+
+[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+
+### How accessible this website is
+
+We know some parts of this website are not fully accessible.
+
+- some headings do not fit with the content that follows
+- some links do not clearly explain where they lead, or that they lead to external sites
+- there is some inaccessible language
+- some tables are structured in an inaccessible way
+- there are some links to documents in PDF format that are not accessible
+
+### What to do if you cannot access parts of this website
+
+If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille, [select __Report a problem__ on the data.gov.uk Support page](https://data.gov.uk/support) to message the team with details of your request.
+
+We’ll aim to reply within 2 working days.
+
+### Reporting accessibility problems with this website
+
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, [select __Report a problem__ on the data.gov.uk Support page](https://data.gov.uk/support) to message the team with details of your request.
+
+### Enforcement procedure
+
+The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, [the Equality Advisory and Support Service (EASS)][EASS].
```

Comment:
This is an alias link to `EASS` but you haven't defined `EASS` anywhere in the document. You'll need to add `EASS: https://example.com` at the bottom of the doc, or just put the URL inline (`[EASS]` -> `(https://example.com)`) as you have done with other links.

Reply:
cheers @ChrisBAshton - making the commit now

Reply:
commit is made @ChrisBAshton - all look ok from your end?